src/dll.py

```diff
@@ -0,0 +1,84 @@
+class Node(object):
+    """Node class."""
+
+    def __init__(self, data, prev=None, next=None):
+        """Initialize Node."""
+        self.data = data
+        self.prev = prev
+        self.next = next
+
+
+class DoublyLinkedList(object):
+    """Doubly linked list class."""
+
+    def __init__(self, iterable=None):
+        """Intialize DLL."""
+        self.head = None
+        self.tail = None
+        self._size = 0
+        if iterable is not None:
```

Comment:
Note that your tests are not covering lines 20-21, since none of your tests initialize a DLL on an iterable. I don't mind too much, since the assignment doesn't explicitly say that the DLL should be able to initialize with a passed argument.